test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -297,7 +307,12 @@ def initialize(code, message, failed_attempts)
 
     def write_log_entries(_request, _call)
```

Comment:
Is there a reason we're keeping this class as a subclass of `Google::Logging::V2::LoggingServiceV2::Service`, rather than making it a subclass of `Google::Cloud::Logging::V2::LoggingServiceV2Client`, same as `GRPCLoggingMockService`?